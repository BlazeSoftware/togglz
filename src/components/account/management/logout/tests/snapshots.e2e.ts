import snapIt from '../../../../../test/snap-it';

describe('logout', () => {
  snapIt('account-logout')('renders correctly', '<account-logout></account-logout>');
});
