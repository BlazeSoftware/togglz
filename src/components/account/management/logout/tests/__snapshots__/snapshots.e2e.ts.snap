// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logout renders correctly 1`] = `"<account-logout class=\\"hydrated\\"><div class=\\"o-container o-container--small u-window-box-xlarge u-centered\\"><stencil-route-title class=\\"hydrated\\"></stencil-route-title><div style=\\"font-size: 4em;\\">ðŸ‘‹</div><div><h2 class=\\"c-heading\\">Logged out.</h2><p class=\\"c-paragraph u-text--quiet\\">See you soon!</p><p class=\\"c-paragraph\\"><stencil-route-link class=\\"hydrated\\"><!----><a class=\\"c-link\\" href=\\"/\\">Go back to the home page</a></stencil-route-link></p><p class=\\"c-paragraph\\"><stencil-route-link class=\\"hydrated\\"><!----><a class=\\"c-link\\" href=\\"/login\\">Log in again</a></stencil-route-link></p></div></div></account-logout>"`;
